sample/src/main/java/com/karumi/rosie/sample/base/view/fragment/MarvelFragment.java

```diff
@@ -20,7 +20,16 @@
 import com.karumi.rosie.sample.R;
 import com.karumi.rosie.view.RosieFragment;
 
+import butterknife.ButterKnife;
+
 public abstract class MarvelFragment extends RosieFragment {
+
+  @Override
+  protected void onPreparePresenter() {
+    super.onPreparePresenter();
+    ButterKnife.bind(this, getView());
```

Comment:
In the case of the fragment create a new method is not needed but this call to ButterKnife should be moved to ``onCreateView`` method.